dask/dataframe/core.py

```diff
@@ -580,6 +580,56 @@ def clear_divisions(self):
         divisions = (None,) * (self.npartitions + 1)
         return type(self)(self.dask, self._name, self._meta, divisions)
 
+    def compute_current_divisions(self, col=None):
+        """Compute the current divisions of the dataframe.
+        This method triggers immediate computation. If you find yourself running this command
+        repeatedly for the same dataframe, we recommend storing the result
+        so you don't have to rerun it.
+
+        If the column or index values overlap between partitions, raises ``ValueError``.
+        To prevent this, make sure the data are sorted by the column or index.
+        Parameters
```

Comment:
```suggestion
        To prevent this, make sure the data are sorted by the column or index.
        
        Parameters
```
I'm not sure, does Sphinx need empty lines between these sections?

Reply:
It looks better with it anyways